### ISSUE TITLE
FNX-13103 ⁃ [Bug] Regression - Push notifications don't work in Beta 78.0.1

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#12085]
## Steps to reproduce
1. Go to foxpush.com, https://gauntface.github.io/simple-push-demo/
1. Send a push notification
1. Verify the notifications tray
    1. Expected behavior
The notification message is sent.
    1. Actual behavior
No notification sent.
    1. Device information

* Android device: Pixel 4 (Android 10), Huawei P20 (Android 9)
* Fenix version: 78.0.1-beta1. 
Worked before, on Fenix beta 5.2.0. Works in Nightly 6/29

**Comment by sheikh-azharuddin** [comment]
Works fine in latest nightly...maybe they will merge soon in beta

**Comment by jonalmeida** [comment]
We're getting the push message, but we're not seeing any notification:
```
2020-06-29 12:25:38.138 ? D/push_ffi: push_dispatch_info_for_chid
2020-06-29 12:25:38.139 ? D/push_ffi: push_decrypt
```

This might have been a regression from https://github.com/mozilla-mobile/android-components/issues/7161.

**Comment by jonalmeida** [comment]
Found the source of the problem: The fix in https://github.com/mozilla-mobile/android-components/issues/7161 relies on changes in GeckoView 79. It seems like we're using GeckoView 78 here, so we drop all push messages as a result.

We can make a separate patch for GV 78, but maybe we can get 79 soon?

**Comment by liuche** [comment]
yes! This preliminary build is 78.0.1 and made for early QA testing but the primary test build TOMORROW 6/30 will be built with 79, so let's retest this again tomorrow with the new 79.0.0-beta.1 build. Thanks for finding this!

**Comment by sheikh-azharuddin** [comment]
On a side note the notification feature needs to be refined further...for example it does not show full message in the push notification...plus on clicking it does nothing ideally should open the webpage....

**Comment by Dunexus** [comment]
@sheikh-azharuddin https://github.com/mozilla-mobile/fenix/issues/11375 and https://github.com/mozilla-mobile/fenix/issues/11138 cover these two notification issues

**Comment by sv-ohorvath** [comment]
Verified as fixed on Beta 79.0.1.
